drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -445,6 +458,21 @@ def _validate_do_attempt(self):
                 self.i18n_service.gettext("Max number of attempts reached")
             )
 
+    def _validate_show_answer(self):
+        """
+        Validates if `show_answer` handler should be executed
```

Comment:
@arbrandes Nit: Add period to the end of this sentence. Also, it would make sense to replicate info about possible exceptions from L392-L394 here.